dtfs-central-api/api-tests/v1/deals/expectAddedFields.js

```diff
@@ -35,6 +44,13 @@ const expectAddedFields = (obj) => {
   return expectation;
 };
 
+const expectAddedFields = ({ baseDeal, userId, auditRecordType }) => {
+  const expectation = addBaseFields(baseDeal);
+  const expectationWithAuditRecord = addExpectedAuditRecord(expectation, { id: userId, userType: auditRecordType });
```

Comment:
This feels like it would be harder to maintain since its not calling one of the `generateAuditDetails` helpers